geotrek/api/management/commands/sync_mobile.py

```diff
@@ -87,7 +88,7 @@ def sync_view(self, lang, view, name, url='/', params=None, headers={}, zipfile=
         # If new file is identical to old one, don't recreate it. This will help backup
         if os.path.isfile(oldfilename) and filecmp.cmp(fullname, oldfilename):
             os.unlink(fullname)
-            os.link(oldfilename, fullname)
+            shutil.copy(oldfilename, fullname)
```

Comment:
as comment says, os.link is used to don't recreate file and use and [hardlink](https://docs.python.org/3/library/os.html#os.link) 
This save many drive space on sync